swift-package-baseline/breakage-allowlist-path.txt

```diff
@@ -0,0 +1 @@
+API breakage: constructor Intersection.init(location:headings:approachIndex:outletIndex:outletIndexes:approachLanes:usableApproachLanes:preferredApproachLanes:usableLaneIndication:outletRoadClasses:tollCollection:tunnelName:restStop:isUrban:regionCode:outletMapboxStreetsRoadClass:railroadCrossing:trafficSignal:stopSign:yieldSign:) has been removed
```

Comment:
The pull request considers adding new parameters with a default value to the constructor of a struct as a breaking change.